rewards/miner_rewards.csv

```diff
@@ -1,435 +1,1436 @@
+105733200,1
+105820800,1
+105908400,1
+105996000,1
+106083600,1
+106171200,1
+106258800,1
+106346400,1
+106434000,1
+106521600,1
+106609200,1
+106696800,1
+106784400,1
+106872000,1
+106959600,1
+107047200,1
+107134800,1
+107222400,1
+107310000,1
+107397600,1
+107485200,1
+107572800,1
+107660400,1
+107748000,1
+107835600,1
+107923200,1
+108010800,1
+108098400,1
+108186000,1
+108273600,1
+108361200,1
+108448800,1
+108536400,1
+108624000,1
+108711600,1
+108799200,1
+108886800,1
+108974400,1
+109062000,1
+109149600,1
+109237200,1
+109324800,1
+109412400,1
+109500000,1
+109587600,1
+109675200,1
+109762800,1
+109850400,1
+109938000,1
+110025600,1
+110113200,1
+110200800,1
+110288400,1
+110376000,1
+110463600,1
+110551200,1
+110638800,1
+110726400,1
+110814000,1
+110901600,1
+110989200,1
+111076800,1
+111164400,1
+111252000,1
+111339600,1
+111427200,1
+111514800,1
+111602400,1
+111690000,1
+111777600,1
+111865200,1
+111952800,1
+112040400,1
+112128000,1
+112215600,1
+112303200,1
+112390800,1
+112478400,1
+112566000,1
+112653600,1
+112741200,1
+112828800,1
+112916400,1
+113004000,1
+113091600,1
+113179200,1
+113266800,1
+113354400,1
+113442000,1
+113529600,1
+113617200,1
+113704800,1
+113792400,1
+113880000,1
+113967600,1
+114055200,1
+114142800,1
+114230400,1
+114318000,1
+114405600,1
+114493200,1
+114580800,1
+114668400,1
+114756000,1
+114843600,1
+114931200,1
+115018800,1
+115106400,1
+115194000,1
+115281600,1
+115369200,1
+115456800,1
+115544400,1
+115632000,1
+115719600,1
+115807200,1
+115894800,1
+115982400,1
+116070000,1
+116157600,1
+116245200,1
+116332800,1
+116420400,1
+116508000,1
+116595600,1
+116683200,1
+116770800,1
+116858400,1
+116946000,1
+117033600,1
+117121200,1
+117208800,1
+117296400,1
+117384000,1
+117471600,1
+117559200,1
+117646800,1
+117734400,1
+117822000,1
+117909600,1
+117997200,1
+118084800,1
+118172400,1
+118260000,1
+118347600,1
+118435200,1
+118522800,1
+118610400,1
+118698000,1
+118785600,1
+118873200,1
+118960800,1
+119048400,1
+119136000,1
+119223600,1
+119311200,1
+119398800,1
+119486400,1
+119574000,1
+119661600,1
+119749200,1
+119836800,1
+119924400,1
+120012000,1
+120099600,1
+120187200,1
+120274800,1
+120362400,1
+120450000,1
+120537600,1
+120625200,1
+120712800,1
+120800400,1
+120888000,1
+120975600,1
+121063200,1
+121150800,1
+121238400,1
+121326000,1
+121413600,1
+121501200,1
+121588800,1
+121676400,1
+121764000,1
+121851600,1
+121939200,1
+122026800,1
+122114400,1
+122202000,1
+122289600,1
+122377200,1
+122464800,1
+122552400,1
+122640000,1
+122727600,1
+122815200,1
+122902800,1
+122990400,1
+123078000,1
+123165600,1
+123253200,1
+123340800,1
+123428400,1
+123516000,1
+123603600,1
+123691200,1
+123778800,1
+123866400,1
+123954000,1
+124041600,1
+124129200,1
+124216800,1
+124304400,1
+124392000,1
+124479600,1
+124567200,1
+124654800,1
+124742400,1
+124830000,1
+124917600,1
+125005200,1
+125092800,1
+125180400,1
+125268000,1
+125355600,1
+125443200,1
+125530800,1
+125538055,1
+125538056,0.023999333
+125538057,0
```

Comment:
Can you elaborate on these final lines? You're also missing a newline char at EOF.

Reply:
at this point, we've shut off the blockchain and the universe has imploded. There is no plan after this 120-year point.